### PR TITLE
chore: add `restartPolicy` into sample because this is required prope…

### PR DESCRIPTION
**Pull request by mztnnrt**
##### background

When copy and apply sample code, this error message displayed.

```
The Job "local-hello-v1" is invalid: spec.template.spec.restartPolicy: Required value: valid values: "OnFailure", "Never"
```

### CODE REVIEW

**Review by Aisuko — APPROVED** [review]
LGTM

**Comment by Aisuko** [comment]
Nice catch, it should with a `restartPolicy`.

**Comment by mztnnrt** [comment]
@Aisuko CC: @natasha41575 @SyamSundarKirubakaran
Thanks ! Can we merge this PR ?

**Comment by Aisuko** [comment]
> @Aisuko CC: @natasha41575 @SyamSundarKirubakaran
> Thanks ! Can we merge this PR ?

Sorry, I do not have permission to merge PR. Maybe you can call the member from this project.

**Comment by Aisuko** [comment]
cc @koba1t may you help us here?

**Comment by koba1t** [comment]
@mztnnrt 
Sorry, I don't have that permission too.
You need to wait for the review by the maintainer.
/cc @natasha41575

**Comment by Aisuko** [comment]
> @mztnnrt Sorry, I don't have that permission too. You need to wait for the review by the maintainer. /cc @natasha41575

Thank you.

**Comment by koba1t** [comment]
It looks like you are right.

```
$ kubectl apply -f job.yaml
The Job "hello" is invalid: spec.template.spec.restartPolicy: Required value: valid values: "OnFailure", "Never"
```

Thanks @mztnnrt!

/lgtm
/approve

**Pull request merged**
